examples/BasicOperations/UpdateExpandedTextAd.php

```diff
@@ -0,0 +1,141 @@
+
+/**
+ * This example updates an expanded text ad. To get expanded text ads, run GetExpandedTextAds.php.
+ */
+class UpdateExpandedTextAd
+{
+    private const CUSTOMER_ID = 'INSERT_CUSTOMER_ID_HERE';
+    private const AD_ID = 'INSERT_AD_ID_HERE';
+
+    public static function main()
+    {
+        // Either pass the required parameters for this example on the command line, or insert them
+        // into the constants above.
+        $options = (new ArgumentParser())->parseCommandArguments([
+            ArgumentNames::CUSTOMER_ID => GetOpt::REQUIRED_ARGUMENT,
+            ArgumentNames::AD_ID => GetOpt::REQUIRED_ARGUMENT
+        ]);
+
+        // Generate a refreshable OAuth2 credential for authentication.
+        $oAuth2Credential = (new OAuth2TokenBuilder())->fromFile()->build();
+
+        // Construct a Google Ads client configured from a properties file and the
+        // OAuth2 credentials above.
+        $googleAdsClient = (new GoogleAdsClientBuilder())->fromFile()
+            ->withOAuth2Credential($oAuth2Credential)
+            ->build();
+
+        try {
+            self::runExample(
+                $googleAdsClient,
+                $options[ArgumentNames::CUSTOMER_ID] ?: self::CUSTOMER_ID,
+                $options[ArgumentNames::AD_ID] ?: self::AD_ID
+            );
+        } catch (GoogleAdsException $googleAdsException) {
+            printf(
+                "Request with ID '%s' has failed.%sGoogle Ads failure details:%s",
+                $googleAdsException->getRequestId(),
+                PHP_EOL,
+                PHP_EOL
+            );
+            foreach ($googleAdsException->getGoogleAdsFailure()->getErrors() as $error) {
+                /** @var GoogleAdsError $error */
+                printf(
+                    "\t%s: %s%s",
+                    $error->getErrorCode()->getErrorCode(),
+                    $error->getMessage(),
+                    PHP_EOL
+                );
+            }
+        } catch (ApiException $apiException) {
+            printf(
+                "ApiException was thrown with message '%s'.%s",
+                $apiException->getMessage(),
+                PHP_EOL
+            );
+        }
+    }
+
+    /**
+     * Runs the example.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     * @param int $adId the ad ID to update
+     */
+    public static function runExample(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        int $adId
+    ) {
+        // Creates an ad with the specified resource name and other changes.
+        $ad = new Ad([
+            'resource_name' => ResourceNames::forAd($customerId, $adId),
+            'expanded_text_ad' => new ExpandedTextAdInfo([
+                'headline_part1' => new StringValue(['value' => 'Cruise to Pluto #' . uniqid()]),
+                'headline_part2' => new StringValue(['value' => 'Tickets on sale now'])
```

Comment:
'description' attribute might be missed:
Golden CL: https://github.com/googleads/google-ads-dotnet/blob/master/examples/BasicOperations/UpdateExpandedTextAd.cs#L81

Reply:
Nice catch. I followed Python. Just pinged Fei to fix this too.